src/main/java/edu/ucsb/cs156/happiercows/jobs/UpdateCowHealthJob.java

```diff
@@ -43,12 +47,22 @@ public void accept(JobContext ctx) throws Exception {
                 Integer totalCows = commonsRepository.getNumCows(commons.getId()).orElseThrow(()->new RuntimeException("Error calling getNumCows(" + commons.getId() + ")"));
 
                 for (UserCommons userCommons : allUserCommons) {
+                    double totalHealths = 0d;
```

Comment:
I'm trying to see if I understand the code here, and if so, whether we might be able to do something with variable names here to help future developers understand the code better.

I think what's happening is that as you update cow health, you are keeping a sum of the new health level of all of the cows, so that in the end, you can set one number for the user commons that represents the average health of all cows owned by this farmer.  Is that correct?

If so, perhaps this might be:
```suggestion
                    double sumNewHealthValueAllCows = 0d;
```

I'm not sure that `totalHealths` conveys that clearly.   Health is not a "countable noun", so having it be plural is just confusing at first.

More below.

Reply:
Agreed, a longer variable name is more clear here! I am currently pushing changes!